bigframes/dataframe.py

```diff
@@ -2074,6 +2074,60 @@ def pivot(
     ) -> DataFrame:
         return self._pivot(columns=columns, index=index, values=values)
 
+    def pivot_table(
+        self,
+        values: typing.Optional[
+            typing.Union[blocks.Label, Sequence[blocks.Label]]
+        ] = None,
+        index: typing.Optional[
+            typing.Union[blocks.Label, Sequence[blocks.Label]]
+        ] = None,
+        columns: typing.Union[blocks.Label, Sequence[blocks.Label]] = None,
+        aggfunc: str = "mean",
+    ) -> DataFrame:
+        if isinstance(index, blocks.Label):
+            index = [index]
+        else:
+            index = list(index)
```

Comment:
Does this have the correct behavior for tuples?  Tuples are tricky because they can be interpreted as individual labels or list of labels.